v2/config/samples/insights/v1alpha1api20180501preview_webtest.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: insights.azure.com/v1alpha1api20180501preview
+kind: WebTest
+metadata:
+  name: samplewebtest
+  namespace: default
+spec:
+  tags:
+    "hidden-link:/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/aso-sample-rg/providers/microsoft.insights/components/sampleappinsights": Resource
```

Comment:
Weird, so the link between the test and the top-level component is by the test being tagged for the component?